### PR TITLE
libPhoneNumberShortNumber target created

### PR DESCRIPTION
**Pull request by rastaarh**
Short Number functionalities + metadata have been split to its own target with corresponding podspec file. Short Number-related tests have been removed from libPhoneNumberTests and moved to libPhoneNumberShortNumberTests

### CODE REVIEW

File: libPhoneNumber/NBMetadataHelper.m [hunk 1]
```diff
@@ -9,18 +9,12 @@
 #import "NBMetadataHelper.h"
 #import "NBGeneratedPhoneNumberMetaData.h"
 #import "NBPhoneMetaData.h"
-
+#import "NBGeneratedShortNumberMetadata.h"
 @interface NBMetadataHelper ()
```
Comment (paween) [inline libPhoneNumber/NBMetadataHelper.m:13]: Add a new line before @interface

File: libPhoneNumberShortNumber.podspec [hunk 2]
```diff
@@ -0,0 +1,22 @@
+Pod::Spec.new do |s|
+  s.name         = "libPhoneNumberGeocoding"
```
Comment (paween) [inline libPhoneNumberShortNumber.podspec]: libPhoneNumberShortNumber

File: libPhoneNumberShortNumber.podspec [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+  s.version      = "0.9.15"
```
Comment (paween) [inline libPhoneNumberShortNumber.podspec]: Just make this 1.0.0

File: libPhoneNumberShortNumber.podspec [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+  s.summary      = "Short Number Support for libPhoneNumber-iOS"
+  s.description  = <<-DESC
+libPhoneNumberShortNumber for iOS
+iOS library for implementing libPhoneNumber features on short numbers. 
+DESC
+  s.homepage     = "https://github.com/iziz/libPhoneNumber-iOS.git"
+  s.license      = 'Apache License, Version 2.0 (http://www.apache.org/licenses/LICENSE-2.0)'
+  s.authors      = { "rastaarh" => "rastaar@google.com"}
```
Comment (paween) [inline libPhoneNumberShortNumber.podspec]: Maybe add me there too so when you leave Google after the internship, people can still contact someone at the company

**Review by paween — COMMENTED** [review]
Make sure to run clang format as well.

File: libPhoneNumberShortNumber.podspec (hunk 2 above)
Comment (rastaarh) [reply libPhoneNumberShortNumber.podspec]: done.

File: libPhoneNumberShortNumber.podspec (hunk 3 above)
Comment (rastaarh) [reply libPhoneNumberShortNumber.podspec]: done.

File: libPhoneNumberShortNumber.podspec (hunk 4 above)
Comment (rastaarh) [reply libPhoneNumberShortNumber.podspec]: yup, done.

File: libPhoneNumber/NBMetadataHelper.m (hunk 1 above)
Comment (rastaarh) [reply libPhoneNumber/NBMetadataHelper.m:13]: yup done.

**Comment by rastaarh** [comment]
fixed a minor merge conflict. Made all edits from code review above.

**Pull request merged**